### PR TITLE
pd-control: update config of balance-hot-region-scheduler

### PR DESCRIPTION
**Pull request by lhy1024**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)
- [ ] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Review by Yisaer — APPROVED** [review]

**Review by HunDunDM — APPROVED** [review]

File: pd-control.md [hunk 1]
```diff
@@ -1076,13 +1076,13 @@ Encoding 格式示例：
     >> scheduler config balance-hot-region-scheduler set src-tolerance-ratio 1.1
     ```
 
-- `read-priorities`、`write-leader-priorities`、`write-peer-priorities` 用于控制处理不同类型的热点时，优先均衡的第一维度和第二维度。对于 `read` 和 `write-leader` 类型的热点，可选的维度有 `query`、`byte` 和 `key`。对于 `write-peer` 类型的热点，可选的维度有 `byte` 和 `key`。若集群组件未全部升级到 v5.2 及以上版本，这些配置不会生效，固定使用兼容配置。通常用户不需要修改这些配置项。
+- `read-priorities`、`write-leader-priorities`、`write-peer-priorities` 用于控制处理不同类型的热点时，优先均衡的第一维度和第二维度。对于 `read` 和 `write-leader` 类型的热点，可选的维度有 `query`、`byte` 和 `key`。对于 `write-peer` 类型的热点，可选的维度有 `byte` 和 `key`。若集群组件未全部升级到 v5.2 及以上版本，这些配置不会生效，固定使用兼容配置，升级完成后会继续保持兼容配置。兼容配置与新集群的默认配置的区别是不含有 `query` 维度。通常用户不需要修改这些配置项。
```
Comment (Liuxiaozhen12) [inline pd-control.md:1079]:
```suggestion
- `read-priorities`、`write-leader-priorities`、`write-peer-priorities` 用于控制处理不同类型的热点时，优先均衡的第一维度和第二维度。对于 `read` 和 `write-leader` 类型的热点，可选的维度有 `query`、`byte` 和 `key`。对于 `write-peer` 类型的热点，可选的维度有 `byte` 和 `key`。若集群组件未全部升级到 v5.2 及以上版本，这些配置不会生效，固定使用兼容配置。集群组件全部升级完成后，会继续保持兼容配置。兼容配置与新集群的默认配置的区别是，兼容配置没有 `query` 维度。通常用户不需要修改这些配置项。
```

File: pd-control.md [hunk 2]
```diff
@@ -1076,13 +1076,13 @@ Encoding 格式示例：
...
 
     ```bash
     >> scheduler config balance-hot-region-scheduler set read-priorities query,byte
     ```
 
-- `strict-picking-store` 是控制热点调度搜索空间的开关，打开时会在保证稳定性的前提下进行热点调度。通常为打开，关闭后只保证第一优先级维度的均衡度，可能会导致其他维度的均衡度降低。通常用户不需要修改这个配置项。
+- `strict-picking-store` 是控制热点调度搜索空间的开关，打开时热点调度的目标是保证所配置的多个维度的均衡，关闭后只保证处于第一优先级的维度的均衡。通常为打开，关闭后能让处于第一优先级的维度的均衡表现更好，但可能会导致其他维度不再那么均衡。通常用户不需要修改这个配置项。
```
Comment (Liuxiaozhen12) [inline pd-control.md:1085]: 这里同时紧挨着出现两个“关闭后”，可否删除后一句或者将两句合并一下呢？

File: pd-control.md (hunk 1 above)
Comment (qiancai) [inline pd-control.md:1079]:
```suggestion
- `read-priorities`、`write-leader-priorities`、`write-peer-priorities` 用于控制调度器从哪些维度进行热点均衡。

    - `read-priorities`、`write-leader-priorities` 用于控制调度器在处理 read 和 write-leader 类型的热点时优先均衡的维度，支持配置两个维度。可选的维度有 `query`、`byte` 和 `key`。
    - `write-peer-priorities` 用于控制调度器在处理 write-peer 类型的热点时优先均衡的维度，支持配置 `byte` 和 `key` 维度。
    
    > ** 注意：**
    >
    > 若集群的所有组件未全部升级到 v5.2 及以上版本，`query` 维度的配置不生效，部分组件升级完成后调度器仍默认从 `byte` 和 `key` 维度进行热点均衡。通常不需要修改这些配置项。
```

**Review by qiancai — COMMENTED** [review]
@lhy1024

**Comment by lhy1024** [comment]
PTAL @qiancai @Liuxiaozhen12

File: pd-control.md [hunk 3]
```diff
@@ -1076,13 +1076,20 @@ Encoding 格式示例：
     >> scheduler config balance-hot-region-scheduler set src-tolerance-ratio 1.1
     ```
 
-- `read-priorities`、`write-leader-priorities`、`write-peer-priorities` 用于控制处理不同类型的热点时，优先均衡的第一维度和第二维度。对于 `read` 和 `write-leader` 类型的热点，可选的维度有 `query`、`byte` 和 `key`。对于 `write-peer` 类型的热点，可选的维度有 `byte` 和 `key`。若集群组件未全部升级到 v5.2 及以上版本，这些配置不会生效，固定使用兼容配置。通常用户不需要修改这些配置项。
+- `read-priorities`、`write-leader-priorities`、`write-peer-priorities` 用于控制调度器优先从哪些维度进行热点均衡。
```
Comment (Liuxiaozhen12) [inline pd-control.md:1079]:
```suggestion
- `read-priorities`、`write-leader-priorities` 和 `write-peer-priorities` 用于控制调度器优先从哪些维度进行热点均衡。
```

**Review by Liuxiaozhen12 — APPROVED** [review]

File: pd-control.md [hunk 4]
```diff
@@ -1076,13 +1076,20 @@ Encoding 格式示例：
     >> scheduler config balance-hot-region-scheduler set src-tolerance-ratio 1.1
     ```
 
-- `read-priorities`、`write-leader-priorities`、`write-peer-priorities` 用于控制处理不同类型的热点时，优先均衡的第一维度和第二维度。对于 `read` 和 `write-leader` 类型的热点，可选的维度有 `query`、`byte` 和 `key`。对于 `write-peer` 类型的热点，可选的维度有 `byte` 和 `key`。若集群组件未全部升级到 v5.2 及以上版本，这些配置不会生效，固定使用兼容配置。通常用户不需要修改这些配置项。
+- `read-priorities`、`write-leader-priorities` 和 `write-peer-priorities` 用于控制调度器优先从哪些维度进行热点均衡。
+	
+	    - `read-priorities`、`write-leader-priorities` 用于控制调度器在处理 read 和 write-leader 类型的热点时优先均衡的维度，支持配置两个维度。可选的维度有 `query`、`byte` 和 `key`。
```
Comment (Liuxiaozhen12) [inline pd-control.md:1081]: 这里的缩进好像还有点问题，PTAL~

Comment (Liuxiaozhen12) [inline pd-control.md:1081]:
```suggestion
	    - `read-priorities` 和 `write-leader-priorities` 用于控制调度器在处理 read 和 write-leader 类型的热点时优先均衡的维度，支持配置两个维度。可选的维度有 `query`、`byte` 和 `key`。
```

File: pd-control.md [hunk 5]
```diff
@@ -1076,13 +1076,20 @@ Encoding 格式示例：
     >> scheduler config balance-hot-region-scheduler set src-tolerance-ratio 1.1
     ```
 
-- `read-priorities`、`write-leader-priorities`、`write-peer-priorities` 用于控制处理不同类型的热点时，优先均衡的第一维度和第二维度。对于 `read` 和 `write-leader` 类型的热点，可选的维度有 `query`、`byte` 和 `key`。对于 `write-peer` 类型的热点，可选的维度有 `byte` 和 `key`。若集群组件未全部升级到 v5.2 及以上版本，这些配置不会生效，固定使用兼容配置。通常用户不需要修改这些配置项。
+- `read-priorities`、`write-leader-priorities` 和 `write-peer-priorities` 用于控制调度器优先从哪些维度进行热点均衡。
+
+    - `read-priorities` 和 `write-leader-priorities` 用于控制调度器在处理 read 和 write-leader 类型的热点时优先均衡的维度，支持配置两个维度。可选的维度有 `query`、`byte` 和 `key`。
+    - `write-peer-priorities` 用于控制调度器在处理 write-peer 类型的热点时优先均衡的维度，支持配置两个维度，支持配置 `byte` 和 `key` 维度。
```
Comment (qiancai) [inline pd-control.md:1082]:
```suggestion
    - `write-peer-priorities` 用于控制调度器在处理 write-peer 类型的热点时优先均衡的维度，支持配置 `byte` 和 `key` 这两个维度。
```

File: pd-control.md [hunk 6]
```diff
@@ -1076,13 +1076,20 @@ Encoding 格式示例：
...
+    
+    > **注意：**
+    >
+    > 若集群的所有组件未全部升级到 v5.2 及以上版本，`query` 维度的配置不生效，部分组件升级完成后调度器仍默认优先从 `byte` 和 `key` 维度进行热点均衡，集群组件全部升级完成后，也会继续保持该配置。通常用户不需要修改这些配置项。
```
Comment (qiancai) [inline pd-control.md:1086]:
```suggestion
    > 若集群的所有组件未全部升级到 v5.2 及以上版本，`query` 维度的配置不生效，部分组件升级完成后调度器仍默认优先从 `byte` 和 `key` 维度进行热点均衡，集群的所有组件全部升级完成后，也会继续保持该配置。通常用户不需要修改这些配置项。
```

File: pd-control.md [hunk 7]
```diff
@@ -1076,13 +1076,20 @@ Encoding 格式示例：
     >> scheduler config balance-hot-region-scheduler set src-tolerance-ratio 1.1
     ```
 
-- `read-priorities`、`write-leader-priorities`、`write-peer-priorities` 用于控制处理不同类型的热点时，优先均衡的第一维度和第二维度。对于 `read` 和 `write-leader` 类型的热点，可选的维度有 `query`、`byte` 和 `key`。对于 `write-peer` 类型的热点，可选的维度有 `byte` 和 `key`。若集群组件未全部升级到 v5.2 及以上版本，这些配置不会生效，固定使用兼容配置。通常用户不需要修改这些配置项。
+- `read-priorities`、`write-leader-priorities` 和 `write-peer-priorities` 用于控制调度器优先从哪些维度进行热点均衡，支持配置两个维度.
```
Comment (TomShawn) [inline pd-control.md:1079]:
```suggestion
- `read-priorities`、`write-leader-priorities` 和 `write-peer-priorities` 用于控制调度器优先从哪些维度进行热点均衡，支持配置两个维度。
```

**Review by qiancai — APPROVED** [review]
LGTM

**Pull request merged**